mini-wallet.md

```diff
@@ -52,86 +50,106 @@ Options:
-dmw test --help
-Usage: dmw test [OPTIONS]
-
-Options:
-  -t, --target TEXT               Target mini-wallet application URL.
-                                  [default: http://localhost:8888]
-
-  -h, --stub-bind-host TEXT       The host the miniwallet stub server will
-                                  bind to  [default: localhost]
-
-  -p, --stub-bind-port INTEGER    The port the miniwallet stub server will
-                                  bind to. Random if empty.
+### How it works
 
-  -u, --stub-diem-account-base-url TEXT
-                                  The address that will be used for offchain
-                                  callbacks. Defaults to
-                                  http://localhost:{random_port}
+The `diem.testing.suites` package contains the Diem MiniWallet Test Suite and includes the pytest file `conftest.py`.
 
-  -j, --jsonrpc TEXT              Diem fullnode JSON-RPC URL.  [default:
-                                  http://testnet.diem.com/v1]
-
-  -f, --faucet TEXT               Testnet faucet URL.  [default:
-                                  http://testnet.diem.com/mint]
+Launch a pytest runner to run tests using:
 
-  --pytest-args TEXT              Additional pytest arguments, split by empty
-                                  space, e.g. `--pytest-args '-v -s'`.
+`dmv test`
 
-  -d, --test-debug-api BOOLEAN    Run tests for debug APIs.  [default: False]
-  -v, --verbose                   Enable verbose log output.  [default: False]
-  --help                          Show this message and exit.
-```
+The `conftest.py` starts a stub MiniWallet as a counterparty service for testing payment with the target server specified by the --target option here.
 
-### How it works
 
-1. The test suite is located in the `diem.testing.suites` package and includes the pytest file `conftest.py`.
-2. `dmw test` launches a pytest runner to run tests.
-3. The `conftest.py` starts a stub MiniWallet as a counterparty service for testing payment with the target server specified by the `--target` option.
 
+### Work with your local test network
 
-### Working with a local testnet
+The Diem MiniWallet server connects to the Diem testnet by default. However, you can use the MiniWallet to test your wallet application in your own local test network as well.
 
-1. [Download](https://docs.docker.com/get-docker/) and install Docker and Docker Compose (comes with Docker for Mac and Windows).
-2. Download Diem testnet docker compose config: https://github.com/diem/diem/blob/master/docker/compose/validator-testnet/docker-compose.yaml
-3. Run `docker-compose -f <path to your validator-testnet/docker-compose.yaml file> up --detach`
-   * A Faucet will be available at http://127.0.0.1:8000
-   * A JSON-RPC server will be available at http://127.0.0.1:8080
-4. Test your application with a local testnet: `dmw test --jsonrpc http://127.0.0.1:8080 --faucet http://127.0.0.1:8000 --target http://localhost:9999`
+To run your local test network:
```

Comment:
"testnet" has special meaning. "test network" is not same with "testnet"

Reply:
changed